app/locale/pt-BR.coffee

```diff
@@ -1594,8 +1594,8 @@ module.exports = nativeDescription: "Português (Brasil)", englishDescription: "
 #    no_new_students_imported: "No new students imported"
 #    error_in_importing_classrooms: "Error in importing classrooms"
 #    error_in_importing_students: "Error in importing students"
-#    classroom_items_description: "Whether students should earn gems and equip items during gameplay."
-#    classroom_live_completion: "Whether to enable coding autocomplete in this classroom. Disabled: turns autocomplete off for all levels. Enabled: allows students to choose. We recommend leaving this enabled."
+    classroom_items_description: "Se os alunos devem ganhar gemas e equipar itens durante o jogo."
```

Comment:
indentation/spacing should be similar to `licenses_needed: "Licenças necessárias"` string above. Adding a space here and the line below should get that.